core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -203,6 +204,7 @@ object Controller {
     // if deploying multiple instances (scale out), must pass the instance number as the
     require(args.length >= 1, "controller instance required")
     val instance = args(0).toInt
+    OpenTracingProvider.apply("Controller")
```

Comment:
Instead of having controller and invoker initialize the tracer it would be better to capture the name as part of config itself. Then you can specify the name in reference.conf of invoker and controller and no need for any explicit initialization call